parking_permits/admin_resolvers.py

```diff
@@ -327,19 +327,21 @@ def resolve_vehicle(obj, info, reg_number, national_id_number):
 def update_or_create_address(address_info):
     if not address_info:
         return None
-    location = Point(*address_info["location"], srid=settings.SRID)
-    address_obj = Address.objects.update_or_create(
-        street_name=address_info["street_name"],
-        street_name_sv=address_info["street_name_sv"]
-        if address_info.get("street_name_sv")
-        else "",
-        street_number=address_info["street_number"],
-        city=address_info["city"] if address_info.get("city") else "",
-        city_sv=address_info["city_sv"] if address_info.get("city_sv") else "",
+    address, _ = Address.objects.update_or_create(
```

Comment:
We are already importing `gettext_lazy as _` so it is better to use some other variable name here to avoid potential hard-trackable issues or bugs in the future.